data/data_replication/json_cleaning_ACM.py

```diff
@@ -0,0 +1,78 @@
+import json
+from nameparser import HumanName
+
+with open('ACM.json') as f:
+  data = json.load(f)
+
+# for each item, delete unnecessary author info
+for i in range(205):
+    try:
+        for j in range(len(data['documents'][i]['info']['authors']['author'])):
+            del data['documents'][i]['info']['authors']['author'][j]['@pid']
+    except KeyError:
+        pass 
```

Comment:
I'm not at all surprised that there are instances when we have missing data, but could we log this to alert the user rather than `pass` on the `KeyError`?

Reply:
Will do!